proxy/proxy.go

```diff
@@ -966,8 +966,7 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 				status = http.StatusServiceUnavailable
 			}
 			p.tracing.setTag(ctx.proxySpan, HTTPStatusCodeTag, uint16(status))
-			//lint:ignore SA1019 Temporary is deprecated in Go 1.18, but keep it for now (https://github.com/zalando/skipper/issues/1992)
-			return nil, &proxyError{err: fmt.Errorf("net.Error during backend roundtrip to %s: timeout=%v temporary='%v': %w", req.URL.Host, nerr.Timeout(), nerr.Temporary(), err), code: status}
+			return nil, &proxyError{err: fmt.Errorf("net.Error during backend roundtrip to %s: timeout=%v : %w", req.URL.Host, nerr.Timeout(), err), code: status}
```

Comment:
Redundant whitespace before colon: `timeout=%v : %w"`

Reply:
Also maybe we can change `net.Error during backend roundtrip ...` to `%T during backend roundtrip ...` to get e.g. this output `*net.DNSError during backend roundtrip ...`

https://go.dev/play/p/QyCCO_4d5yL

Reply:
Nice idea with %T 🙂 Fixed in the next patchset!